Source/WebCore/inspector/InspectorStyleSheet.cpp

```diff
@@ -1436,23 +1471,90 @@ Ref<Protocol::CSS::CSSStyle> InspectorStyleSheet::buildObjectForStyle(CSSStyleDe
     return result;
 }
 
-ExceptionOr<void> InspectorStyleSheet::setStyleText(const InspectorCSSId& id, const String& text, String* oldText)
+static inline bool isNotSpaceOrTab(UChar character)
```

Comment:
NIT: i kinda feel like this should be flipped (i.e. `isSpaceOrTab`) and the callsites do a `!` as that'd be easier to read (and likely more composable with other checks if needed in the future)

Reply:
That doesn't quite work since this is a function we pass to String::find to find the first character that isn't a space or tab...